ZitiNativeApiForDotnetCore/msvc-build.bat

```diff
@@ -32,15 +32,23 @@ REM copy library/ZitiStatus.cs ../Ziti.NET.Standard/src/OpenZiti
 
 ECHO Build from cmake using: 
 ECHO     cmake --build %BUILDFOLDER%\x86 --config Debug
+cmake --build %BUILDFOLDER%\x86 --config Debug
 ECHO     cmake --build %BUILDFOLDER%\x86 --config Release
 cmake --build %BUILDFOLDER%\x86 --config Release
 ECHO. 
 ECHO     cmake --build %BUILDFOLDER%\x64 --config Debug
+cmake --build %BUILDFOLDER%\x64 --config Debug
 ECHO     cmake --build %BUILDFOLDER%\x64 --config Release
 cmake --build %BUILDFOLDER%\x64 --config Release
 ECHO. 
 ECHO Or open %BUILDFOLDER%\ziti-sdk.sln
 
+REM install nuget
```

Comment:
i considered removing the entire file tbh - but let's revert these changes now too

Reply:
done